daliuge-translator/test/dropmake/logical_graphs/testLoop.graph

```diff
@@ -45,7 +45,7 @@
                     "defaultValue": "",
                     "description": "",
                     "name": "num_of_iter",
-                    "persist": false,
```

Comment:
Whoops I must have accidentally changed this last week accidentally :sweat_smile: